app/views/questions/show.html.erb

```diff
@@ -1,10 +1,14 @@
 <%= javascript_include_tag('comment_expand') %>
 <%= javascript_include_tag('notes') %>
 <%= javascript_include_tag('textbox_expand') %>
-<script> var comments_length = <%= @node.comments.length %>; $(function(){
-  $E.initialize();
-  $("img").lazyload();
-});</script>
+<script> 
+  var comments_length = <%= @node.comments.length %>; 
```

Comment:
This is not getting used anywhere, right?

Reply:
`comments_length`? I think it still is here:
https://github.com/publiclab/plots2/blob/97247e30289dfb27f93446b84d036ad9811e7983/app/assets/javascripts/question.js#L13-L23

Reply:
But I don't know what this code does 😅

Reply:
This looks like old code related to the "Answers" system which we've deprecated; answers were back turned into comments. It seems it was for showing only the first... 3 answers, hiding the rest behind a "show more" kind of thing? I think that's all gone now, so we should be good to remove it.